src/app/core/admin/user-management/admin-list-users.component.ts

```diff
@@ -150,15 +155,18 @@ export class AdminListUsersComponent extends AbstractPageableDataComponent<User>
 			});
 
 		this.sortEvent$.next(this.headers.find((header: any) => header.default) as SortChange);
-
-		super.ngOnInit();
 	}
 
 	confirmDeleteUser(user: User) {
 		const id = user.userModel._id;
-		const username = user.userModel.username;
-
-		console.error('Delete User not yet implemented.');
+		this.adminUsersService.removeUser(id).subscribe({
+			next: data => {
+				this.reload();
```

Comment:
@stella-masucci - instead of setting up a second set of subscriptions for the route parameters ( what was moved into the `reload` method ), we should leave that in the `ngOnInit` method and simply call a `this.load$.next(true);` here. That will trigger a reload of the data with all of the current sorting, filtering, and paging functionality.